apps/ledger-live-desktop/src/renderer/components/BalanceInfos/index.tsx

```diff
@@ -114,21 +114,17 @@ export default function BalanceInfos({ totalBalance, valueChange, isAvailable, u
   const { t } = useTranslation();
   const history = useHistory();
 
-  // PTX smart routing feature flag - buy sell live app flag
-  const ptxSmartRouting = useFeature("ptxSmartRouting");
   // Remove "SWAP" and "BUY" redundant buttons when portafolio exchange banner is available
   const portfolioExchangeBanner = useFeature("portfolioExchangeBanner");
   const onBuy = useCallback(() => {
     setTrackingSource("Page Portfolio");
     history.push({
       pathname: "/exchange",
-      state: ptxSmartRouting?.enabled
```

Comment:
If we do not have it, can we add a subtask to remove this firebase flag?

Reply:
yes, I'll remove it once its deployed.